guides/doc-Provisioning_Guide/topics/Cloud-Amazon_EC2.adoc

```diff
@@ -239,6 +239,10 @@ ifeval::["{build}" != "satellite"]
 You can use {ProjectName} finish templates during the provisioning of Linux instances in an Amazon EC2 environment.
 endif::[]
 
+If you want to use a Finish template with SSH, {Project} must reside within the EC2 environment and in the correct security group.
+{Project} currently performs SSH finish provisioning directly, not using {SmartProxyServer}.
+If {ProjectServer} does not reside within EC2, the EC2 virtual machine reports an internal IP rather than the necessary external IP with which it can be reached.
```

Comment:
Are you aware that Asciidoctor will build this as one paragraph because there is not newline between the lines?

Reply:
I am. I left newlines for clarity as I was writing in Atom. It is one paragraph of text in reality. It is not a poem :)

Reply:
Good, merge this I will in this case